.github/workflows/release.yml

```diff
@@ -14,6 +14,8 @@ permissions:
 
 env:
   CARGO_TERM_COLOR: always
+  BIOME_WEBSITE_REPO: biomejs/website
+  BIOME_PUSH_ON_MAIN_EVENT_TYPE: biome-push-on-main-event
```

Comment:
Should this event be named differently? Something like `biome-version-released-event`

Reply:
Yep I also think splitting is a good idea